ledger/financial/cmd/client/mod.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"fmt"
+	"ledger/financial/serde"
+	"net/url"
+	"os"
+
+	"github.com/hyperledger/sawtooth-sdk-go/logging"
+	"github.com/jessevdk/go-flags"
+	flag "github.com/jessevdk/go-flags"
+)
+
+var defaultSerde = serde.JSON{}
+
+const (
+	// BatchSubmitAPI represents the REST endpoint to send batches
+	BatchSubmitAPI string = "batches"
```

Comment:
what are batches in this context ?

Reply:
Batches, [in the Sawtooth sense](https://sawtooth.hyperledger.org/docs/1.2/architecture/transactions_and_batches.html), are a container for transactions that must be applied or rejected in block in case some of them fail. Transactions must always be wrapped in a batch, even for a single transaction.